tests/test_serialization.py

```diff
@@ -296,17 +295,17 @@ def test_decimal_field_special_values_not_permitted(self, user):
 
         field = fields.Decimal(places=2)
 
-        with pytest.raises(ValidationError):
+        with pytest.raises(Exception):
```

Comment:
Wonder if these assertions are worth it at this point 🤔

Reply:
Ya, it's probably not necessary to have regression tests that protect against catching errors again. 😄